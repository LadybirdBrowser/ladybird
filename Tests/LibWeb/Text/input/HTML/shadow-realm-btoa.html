<script src="../include.js"></script>
<script>
    test(() => {
        const realm = new ShadowRealm();
        const result = realm.evaluate(`() => self.btoa('Hello, world!')`)();
        println(`Shadow realm evaluation returned: ${result}`);
    });
</script>
