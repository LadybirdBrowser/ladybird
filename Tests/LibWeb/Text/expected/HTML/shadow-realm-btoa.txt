Shadow realm evaluation returned: SGVsbG8sIHdvcmxkIQ==
